sympy/matrices/common.py

```diff
@@ -2377,7 +2454,9 @@ def __pow__(self, exp):
 
         return self.pow(exp)
 
-    def pow(self, exp, jordan=None):
+
+    def pow(self, exp, jordan=None, cayley=False):
```

Comment:
Having `method='jordan'` is better than two boolean arguments

Reply:
The pow method here is new since the last release of sympy so the signature can be changed.

Reply:
Yes, I’d also want to use a single keyword to control every methods